### DISCUSSION TITLE
Twilio Logs and its pattern

### DISCUSSION THREAD

**Discussion by sang-yeo** [twilio-labs/paste#1621] [Office hours] [answered]
Hello, 
I have worked on [Twilio Logs audit](https://docs.google.com/spreadsheets/d/1ehOaL-P34u7pRntShmjBeMm12s4fvp-883ul-B0H_5Y/edit?usp=sharing) and discovered inconsistent UX patterns throughout Logs. We'd like to share our findings and gather DSYS feedback on the [initial ideas](https://www.figma.com/file/8kTUMFHmpBv2bZW0L7yPci/Peach?node-id=79%3A23185) we came up with.

**Comment by nkrantz** [comment]
Hi @sang-yeo, thanks for starting this Discussion! We'll review ahead of time and discuss in Office Hours today.

**Comment by kate-porter** [comment]
Hey @sang-yeo! As promised, here are those [toggle button / toolbar explorations](https://www.figma.com/file/mgIzaySuGzC79nPCQujvcU/Toolbar-Toggle-Button-Exploration?node-id=1%3A7831) we worked on with the Flex team (you can ignore the dark mode variations) - you can probably borrow some of these styles for your checkbox buttons. Let me know if you have any questions about what we did there!

**Reply by sang-yeo** [reply]
Thank you!

**Comment by serifluous** [comment, marked as answer]
Recording from this office hours session can be found here: https://www.loom.com/share/3006beed70e34f879d380ae1fd9ba4ea

Password is in Slack.
